### DISCUSSION TITLE
命名建议

### DISCUSSION THREAD

**Discussion by QMHTMY** [rbatis/rbatis#280] [General]
英文单词中辅音须配合元音 a, e, i, o, u 或发元音的音节作为发音单元，而 rbatis 中 rb 都是辅音，建议 r 后加一个或两个元音便于发音，也好记忆。rbatis 目前只能读成 r batis，中间有停顿。

下面是建议命名：
rabits      类似兔子发音 rabbits
rebatis   re 表示再的意思，意思是用 rust 再造 mybatis
ribatis
robatis
rubatis

以上都是我个人的建议，作者可以参考。当然，不改也是没问题的。

**Comment by zhuxiujia** [comment]
感谢建议。 rbatis 目前只能读成 r batis。 不考虑修改库名称。
